src/pages/Delivery/Delivery.layout.jsx

```diff
@@ -66,12 +66,12 @@ const Delivery = ({
           >
             <img src={gps} width="22px" height="31px" alt="Gps icon" style={{ margin: '10px'}} />
           </FooterMapAddress>
-          <FooterMapAddress cols={'0 0 50%'} txt={'Tiempo estimado de llegada'}>
-            <Text medium bold margin="8" style={{ textTransform: 'lowercase', color:'#003B62',fontWeight:'700'}}>
+          <FooterMapAddress style={{fontSize:'25px'}} cols={'0 0 50%'} txt={'Tiempo estimado de llegada'}>
+            <Text medium bold margin="8" style={{ textTransform: 'lowercase',fontSize:'23px', color:'#003B62',fontWeight:'700'}}>
```

Comment:
please left enough space to do more readable your code
```suggestion
            <Text medium bold margin="8" style={{ textTransform: 'lowercase', fontSize: '23px',  color: '#003B62', fontWeight: '700' }}>
```